bld/unit_testers/build-namelist_test.pl

```diff
@@ -163,9 +163,21 @@ sub cat_and_create_namelistinfile {
 #
 # Figure out number of tests that will run
 #
+<<<<<<< HEAD
+my $ntests = 1863;
+||||||| 17e2acb6a
+my $ntests = 1846;
+=======
 my $ntests = 1975;
+>>>>>>> escomp/master
 if ( defined($opts{'compare'}) ) {
+<<<<<<< HEAD
```

Comment:
I somehow missed a couple of conflicts here. For now I will use the value that I see in escomp/master; however, I wonder whether the exice branch added tests that I need to account for in the final numbers...

Reply:
@ekluzek suggested the following approach:
1) Add the difference between HEAD and starting point to MAIN.
2) Run the tests and look at the output to confirm how many tests ran and how many should have run.

Reply:
I tried ./build-namelist_test.pl and it finished with this message:
```
Successfully ran all testing for build-namelist

Cleanup files created
/bin/rm: cannot remove 'env_run.xml': No such file or directory
/bin/rm: cannot remove 'user_nl_ctsm_real_parameters': No such file or directory
/bin/rm: cannot remove 'temp_file.txt': No such file or directory
```
I did not keep track of other output that scrolled past me on the screen. I think that this result is fine based on our conversation this morning @ekluzek but pls let me know if I should investigate further. Thanks!

Reply:
Yep, this is all good, so resolving.